doc/api/fs.md

```diff
@@ -2745,12 +2745,31 @@ changes:
                  it will emit a deprecation warning.
 -->
 
-* `path` {string|Buffer|URL}
+* `path` {string|Buffer|URL} File name or file descriptor
```

Comment:
Can this really be a file descriptor? Also, "file" is not accurate, is it? This could be a symlink or a directory, couldn't it? I think "path" is probably the best description, so I don't know that this needs a text explanation.